src/System.Security.Cryptography.Algorithms/pkg/System.Security.Cryptography.Algorithms.pkgproj

```diff
@@ -5,9 +5,12 @@
     <ProjectReference Include="..\ref\4.0\System.Security.Cryptography.Algorithms.csproj">
       <SupportedFramework>net46</SupportedFramework>
     </ProjectReference>
-    <ProjectReference Include="..\ref\System.Security.Cryptography.Algorithms.csproj">
+    <ProjectReference Include="..\ref\4.1\System.Security.Cryptography.Algorithms.csproj">
       <SupportedFramework>net461;netcore50;netstandardapp1.5</SupportedFramework>
     </ProjectReference>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.Algorithms.csproj">
+      <SupportedFramework>net463;netcore50;netstandardapp1.6</SupportedFramework>
+    </ProjectReference>
```

Comment:
I'm not sure netcore50 should be listed on both 4.1 and (current)

@ericstj, @chcosta How'd we do here?